themes/luci-theme-bootstrap/luasrc/view/themes/bootstrap/header.htm

```diff
@@ -59,6 +59,16 @@ <h4><%:No password set!%></h4>
 				</div>
 			<%- end -%>
 
+			<%- if boardinfo.initramfs -%>
+				<div class="alert-message warning">
+					<h4><%:Booted from read-only partition!%></h4>
```

Comment:
This is slightly misleading, as there indeed is that case as well, rootfs(==squashfs) being mounted, but read-only due to rootfs_data being unavailable and then you end up with a tmpfs overlay. This is a similar situation (ie. changes will also not be stored) but it is different in that the system has not intentionally been started from initramfs, which is what the `initramfs` flag in `system->board` indicates.

Reply:
How would you rephrase this? Could you suggest a sentence?

Reply:
"System running in recovery (initramfs) mode."
I'm aware that not everybody will know what that means, but if it's a visible warning, people will consult search engines in case they don't know themselves. In any way, I think it's important to not create confusion, and as this message has nothing to do with "partition" nor "read-only" (tmpfs is read-write) it would only set people on the wrong track.